common/app/contentapi/ContentApiClient.scala

```diff
@@ -75,7 +75,6 @@ object QueryDefaults {
 trait ApiQueryDefaults extends GuLogging {
 
   def item(id: String): ItemQuery = CapiContentApiClient.item(id)
-  def search: SearchQuery = CapiContentApiClient.search
```

Comment:
This definition was only being used by the `search(edition: Edition)` method (from my testing), we can't keep both once they have the same signature